scripts/ci/exports/utils/module.mjs

```diff
@@ -13,6 +13,12 @@ import { Declaration } from './declaration.mjs';
 import { ExternalModule } from './externalmodule.mjs';
 import { isInternalNode } from './misc.mjs';
 import { createExportResolutionError } from './errorutils.mjs';
+import { globSync } from 'glob';
+import { CKEDITOR5_ROOT_PATH } from '../../../constants.mjs';
+import { PACKAGES_DIRECTORY } from '../../../release/utils/constants.mjs';
+
+// E.g. `packages/ckeditor5-core/` or `packages/ckeditor5/`.
```

Comment:
Since the `mark: true` is important, I would write it down:

```suggestion
// E.g. `packages/ckeditor5-core/` or `packages/ckeditor5/` with a trailing `/` to mark the end of a package name.
```

Reply:
Adressed in [4b24bd2](https://github.com/ckeditor/ckeditor5/pull/19147/commits/4b24bd2aa39e98a7304fc7d1ffe9acaeb8726a0a).